src/SignalR/common/Http.Connections/test/HttpConnectionDispatcherTests.cs

```diff
@@ -3550,6 +3550,10 @@ public async Task RequestTimeoutDisabledWhenConnected(HttpTransportType transpor
 
             await connection.StartAsync();
 
+            // Easy way to make sure everything is set is to send and receive data over the connection
+            await connection.Transport.Output.WriteAsync(new byte[2]);
```

Comment:
Are two bytes necessary for this or is any number of bytes sufficient?

Reply:
Any number, not sure why I put 2 😆